src/components/ConsoleInput.jsx

```diff
@@ -20,6 +20,9 @@ export default class ConsoleInput extends Component {
         inheritFontStylesFromParentElement(this._editor);
       });
     }
+    if (!this._editor.isFocused()) {
+      this._editor.focus();
+    }
```

Comment:
I don’t think this is quite the logic we want—in this case the console will focus any time props are pushed out (which might not even involve a change to the props it’s getting). A concrete example of bad behavior:

* In the JavaScript project code, add a `console.log()` line
* Open the console
* Make a trivial modification to the JavaScript code
* Observe that the console input is now focused, which is not expected/desirable behavior

I believe the best bet will be to add logic that updates the `ui.requestedFocusedLine` Redux state when the `CLEAR_CONSOLE_ENTRIES` action is consumed—however this will need to wait on #1304 which generalizes `requestedFocusedLine` to work with the console. That PR is approved, I just need to get a clean build with the branch up to date from `master`.

Reply:
Thanks for the feedback. Its transfering the focus to console on change in javascript code. Will have a look at #1304